hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/onset/OnsetChecker.java

```diff
@@ -137,15 +137,15 @@ public ResponseCodeEnum checkTransactionBody(@NonNull final TransactionBody txBo
             return INSUFFICIENT_TX_FEE;
         }
 
-        return checkTimeBox(txnId.transactionValidStart(), txBody.transactionValidDuration());
+        return checkTimeBox(txnId.transactionValidStart(), txBody.transactionValidDurationOrElse(Duration.DEFAULT));
     }
 
     private static boolean isPlausibleTxnFee(final long transactionFee) {
         return transactionFee >= 0;
     }
 
     private static boolean isPlausibleAccount(final AccountID accountID) {
-        return accountID.accountNum().isPresent() && accountID.realmNum() >= 0 && accountID.shardNum() >= 0;
+        return accountID != null && accountID.realmNum() >= 0 && accountID.shardNum() >= 0;
```

Comment:
Should we add accountId.hasAccountNum() here?

Reply:
Whoops, I missed this. Thanks for pointing out!